code/__HELPERS/stat_tracking.dm

```diff
@@ -12,7 +13,8 @@
 
 	. = lines.Join("\n")
 
-/proc/stat_tracking_export_to_file_later(filename, costs, counts)
+// For use with the set_cost defines
```

Comment:
```suggestion
/// For use with the set_cost defines
```

Reply:
similar, //////////// EXCITED GROUPS //////////// is the language of my fathers